week-3/refactor/format-as-12-hours.js

```diff
@@ -3,3 +3,4 @@
 // Store this expression in a variable and reference it twice in the function in the correct place
 
 // Explain why it makes more sense to store this expression in a variable
+// It is easier to use variables because the make the steps look less complicated and confusing and the variables can be reused in different places.
```

Comment:
Right. The main reason we prefer using variables and functions and assigning those to couple lines of code blocks, is because:

1) It makes our code look much more cleaner.

2) Most importantly, it makes our code blocks much more usable. Think about it;

Imagine you have a function, which is 10 lines in total. Whenever you want to use that functionality, would it be easier to write those 10 lines everytime, or would it be easier to capsule those 10 lines inside a function and use that 1 lined function instead whenever you want to run that functionality.

Reply:
yes it makes sense that it makes our code blocks more usable. thank you